src/templates/Article.jsx

```diff
@@ -22,6 +24,21 @@ function Article(props) {
       title
     }
   } = article;
+
+  const pathMap = pathname
+    // remove leading and trailing slash
+    .replace(/^\/([a-z0-9/-]+[^/])\/?$/i, '$1')
+    .split('/');
+
+  var metaTitle = '';
```

Comment:
Why can't we use `title` from the `article`? On line 24

Reply:
Please declare with either `const` or `let`

Reply:
I can't check the value of `title` right now (but if it's based on the page title or the front-matter specified one) we are afraid that it's going to be contextless is some cases.

Here is an [example](https://guide.freecodecamp.org/cplusplus/input_and_output) where the title is `Input and Output` without any mention of the context being C++.

We are going to work on a strategy to build it up from the url path, I think @LulaV14 is close to a solution.